www/js/app.js

```diff
@@ -416,6 +419,16 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         // Do the necessary to enable or disable the Service Worker
         setContentInjectionMode(this.value);
     });
+    document.getElementById('useCanvasElementsCheck').addEventListener('change', function () {
+        if (this.checked) {
+            settingsStore.removeItem('useCanvasElements');
```

Comment:
I don't understand why you remove this setting here, instead of setting it to true.

Reply:
Because we are not allowing user to force it on. The app (or rather, WebpHero) will decide if the option is needed. The user can only disable its use (in which case we store `false` as the parameter value), or else return the setting to auto. I could make this clearer in the UI. In fact, the user will never have a chance to force this setting on (even if permitted) because the app will not even show this UI if it determines that the workaround is not needed).